packages/contracts-bedrock/test/Specs.t.sol

```diff
@@ -807,11 +855,13 @@ contract Specification_Test is CommonTest {
 
     /// @notice Ensures that there's an auth spec for every L1 contract function.
     function testContractAuth() public {
-        string[] memory pathExcludes = new string[](2);
+        string[] memory pathExcludes = new string[](3);
         pathExcludes[0] = "src/dispute/interfaces/*";
         pathExcludes[1] = "src/dispute/lib/*";
-        Abi[] memory abis =
-            ForgeArtifacts.getContractFunctionAbis("src/{L1,dispute,governance,universal/ProxyAdmin.sol}", pathExcludes);
+        pathExcludes[2] = "src/Safe/SafeSigners.sol";
```

Comment:
Why do we need this exclusion? `SafeSigners.sol` only has internal methods so it seems that shouldn't affect the returned array of ABIs?

Reply:
We need it for the same reason we need to ignore the other two paths: this jq command will return with empty output, causing Process.run() to fail.

We could refactor getMethodIdentifiers to handle that for us, but we already have this ability to filter the input to avoid the error.